internal/provider/resource_machine.go

```diff
@@ -64,6 +64,27 @@ func resourceMachine() *schema.Resource {
 				Optional:    false,
 				Required:    false,
 			},
+			"ssh_address": {
+				Description: "The user@host directive for manual provisioning an existing machine via ssh.",
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "",
+				ForceNew:    true,
+			},
+			"public_key": {
+				Description: "The file path to read the public key from.",
```

Comment:
This should be the string containing the key, not the path to the key. See the original proposed resource in the Jira ticket. The idea is that users can load keys using `public_key="xldi3j3jioj..."` or `$file("somewhere.pub")`.